examples/pom.xml

```diff
@@ -32,32 +28,4 @@
         <maven.deploy.skip>true</maven.deploy.skip>
     </properties>
 
-    <build>
```

Comment:
We can drop this, because the coverage plugin is no longer configured by the parent pom in the examples, as they no longer extend it.

Reply:
👍